src/app/space/app-launcher/services/app-launcher-mission-runtime.service.ts

```diff
@@ -75,403 +73,403 @@ export class AppLauncherMissionRuntimeService implements MissionRuntimeService {
+  //     'simple key and value pairs into one or more Linux containers while keeping the containers independent ' +
+  //     'of OpenShift.',
+  //     'suggested': false,
+  //     'runtimes': [
+  //       'vert.x',
+  //       'spring-boot',
+  //       'wildfly-swarm'
+  //     ],
+  //     'url': 'https://github.com/fabric8-launcher/ngx-launcher'
+  //   }, {
+  //     'id': 'health-check',
+  //     'name': 'Health Check',
+  //     'description': 'When you deploy an application, its important to know if it is available and if it can ' +
+  //     'start handling incoming requests. Implementing the health check pattern allows you to monitor the health ' +
+  //     'of an application, which includes if an application is available and whether it is able to service ' +
+  //     'requests.',
+  //     'suggested': true,
+  //     'runtimes': [
+  //       'vert.x',
+  //       'spring-boot',
+  //       'wildfly-swarm'
+  //     ],
+  //     'url': 'https://github.com/fabric8-launcher/ngx-launcher'
+  //   }, {
+  //     'id': 'rest-http',
+  //     'name': 'REST API Level 0',
+  //     'description': 'The REST API Level 0 Mission shows how to map business operations to a remote procedure call ' +
+  //     'endpoint over HTTP using a REST framework. This corresponds to Level 0 in the Richardson Maturity Model. ' +
+  //     'Creating an HTTP endpoint using REST and its underlying principles to define your API lets you quickly ' +
+  //     'prototype and design the API flexibly.',
+  //     'suggested': false,
+  //     'runtimes': [
+  //       'vert.x',
+  //       'spring-boot',
+  //       'wildfly-swarm'
+  //     ],
+  //     'url': 'https://github.com/fabric8-launcher/ngx-launcher'
+  //   }, {
+  //     'id': 'rest-http-secured',
+  //     'name': 'Secured',
+  //     'description': 'The Secured booster expands on the REST API Level 0 booster by securing a REST endpoint ' +
+  //     'using Red Hat SSO. Red Hat SSO implements the Open ID Connect protocol that is an extension of the ' +
+  //     'OAuth 2.0 specification and uses it to issue access tokens to provide clients with various access rights ' +
+  //     'to secured resources. Securing an application with SSO enables you to add security to your applications ' +
+  //     'while centralizing the security configuration.',
+  //     'suggested': false,
+  //     'runtimes': [
+  //       'vert.x',
+  //       'spring-boot',
+  //       'wildfly-swarm'
+  //     ],
+  //     'url': 'https://github.com/fabric8-launcher/ngx-launcher',
+  //     'prerequisite': 'While this mission comes with Red Hat SSO pre-configured for demonstration purposes, it does ' +
+  //     'not explain its principles, usage, or configuration. Before using this mission, ensure that you are familiar ' +
+  //     'with the basic concepts related to Red Hat SSO'
+  //   }] as Mission[]);
+  //   return missions;
+  // }
```

Comment:
This mock data can be removed -- we only wanted to save this for the demo in ngx-forge